### DISCUSSION TITLE
DALLE.ipynb needs a fix in order to use torch >= 1.11

### DISCUSSION THREAD

**Discussion by lxt3** [Denis2054/Transformers-for-NLP-2nd-Edition#9] [General]
torch versions < 1.11 are apparently no longer available in colab. Therefore, to run the DALL-E notebook, one needs to add the lines of code suggested by [digitalShaman](https://github.com/openai/DALL-E/issues/76#issuecomment-1229197798) immediately after 

`dec = load_model("https://cdn.openai.com/dall-e/decoder.pkl", dev)`

in order to avoid the _'Upsample' object has no attribute 'recompute_scale_factor'_ error when running `x_stats = dec(z).float()`

The suggestion worked for me.

**Comment by Denis2054** [comment]
Thank you for solution
